File: 2023-2024/sourcecode/mod3/ResourcePatternBasicsJava/app/src/main/java/com/example/resourcepatternbasicsjava/data/api/PlaceholderApi.java
package com.example.resourcepatternbasicsjava.data.api;

import com.example.resourcepatternbasicsjava.data.CommentResponseItem;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface PlaceholderApi {
    @GET("comments")
    Call<List<CommentResponseItem>> comments();
}

File: 2023-2024/sourcecode/mod3/ResourcePatternBasicsKotlin/app/src/main/java/com/example/resourcepatternbasicskotlin/data/api/PlaceholderApi.kt
package com.example.resourcepatternbasicskotlin.data.api

import com.example.resourcepatternbasicskotlin.data.CommentResponseItem
import retrofit2.Response
import retrofit2.http.GET

interface PlaceholderApi {
    @GET("comments")
    suspend fun comments(): Response<List<CommentResponseItem>>
}